ansible/roles/nomis-weblogic/templates/10.3/u01/app/oracle/Middleware/user_projects/domains/NomisDomain/bin/setDomainEnv.sh

```diff
@@ -394,13 +394,13 @@ if [ "${SERVER_NAME9}" = "WLS_REPOR" ] ; then
 fi
 
 if [ "${SERVER_NAME9}" = "WLS_FORMS" ] ; then
-	EXTRA_JAVA_PROPERTIES="-Xms256m -Xmx512m -XX:PermSize=256m -XX:MaxPermSize=512m -Djava.ext.dirs=/u01/app/oracle/Middleware/forms_home/jdk/jre/lib/ext -Doracle.security.jps.config=/u01/app/oracle/Middleware/user_projects/domains/NomisDomain/config/fmwconfig/jps-config.xml -Doracle.home=/u01/app/oracle/Middleware/forms_home -Ddomain.home=/u01/app/oracle/Middleware/user_projects/domains/NomisDomain -Doracle.instance=/u01/app/oracle/Middleware/forms_instance/ -Doracle.instance.name=forms_instance -Doracle.forms.weblogic=1 -Ddomain.name=NomisDomain ${EXTRA_JAVA_PROPERTIES}"
+	EXTRA_JAVA_PROPERTIES="-Xms256m -Xmx1536m -XX:PermSize=256m -XX:MaxPermSize=512m -Djava.ext.dirs=/u01/app/oracle/Middleware/forms_home/jdk/jre/lib/ext -Doracle.security.jps.config=/u01/app/oracle/Middleware/user_projects/domains/NomisDomain/config/fmwconfig/jps-config.xml -Doracle.home=/u01/app/oracle/Middleware/forms_home -Ddomain.home=/u01/app/oracle/Middleware/user_projects/domains/NomisDomain -Doracle.instance=/u01/app/oracle/Middleware/forms_instance/ -Doracle.instance.name=forms_instance -Doracle.forms.weblogic=1 -Ddomain.name=NomisDomain ${EXTRA_JAVA_PROPERTIES}"
 	export EXTRA_JAVA_PROPERTIES
 	LD_LIBRARY_PATH="/u01/app/oracle/Middleware/forms_home/lib${CLASSPATHSEP}${LD_LIBRARY_PATH}"
 	export LD_LIBRARY_PATH
 fi
 
-EXTRA_JAVA_PROPERTIES=" -Xms512m -Xmx1024m -XX:PermSize=256m -XX:MaxPermSize=512m -Doracle.home=/u01/app/oracle/Middleware/forms_home -Ddomain.home=/u01/app/oracle/Middleware/user_projects/domains/NomisDomain ${EXTRA_JAVA_PROPERTIES}"
+EXTRA_JAVA_PROPERTIES=" -Xms512m -Xmx15364m -XX:PermSize=256m -XX:MaxPermSize=512m -Doracle.home=/u01/app/oracle/Middleware/forms_home -Ddomain.home=/u01/app/oracle/Middleware/user_projects/domains/NomisDomain ${EXTRA_JAVA_PROPERTIES}"
```

Comment:
Is that supposed to be 15364? or should it be 1536?

Reply:
Updated